charts/datree-admission-webhook/values.yaml

```diff
@@ -8,9 +8,9 @@ namespace: ""
 # -- The number of Datree webhook-server replicas to deploy for the webhook.
 replicaCount: 2
 # -- Additional labels to add to all resources.
-customLabels: { }
```

Comment:
that spaces and changes of perttier should not be upload, maybe better just to add the value to skip list and 'save without format'
![image](https://github.com/datreeio/admission-webhook-datree/assets/29673827/5128b546-ef70-494f-912b-6a740afdff85)